stuffer/s2n_stuffer.h

```diff
@@ -73,7 +73,8 @@ extern int s2n_stuffer_reread(struct s2n_stuffer *stuffer);
 extern int s2n_stuffer_rewrite(struct s2n_stuffer *stuffer);
 extern int s2n_stuffer_wipe(struct s2n_stuffer *stuffer);
 extern int s2n_stuffer_wipe_n(struct s2n_stuffer *stuffer, const uint32_t n);
-extern int s2n_stuffer_release_if_empty(struct s2n_stuffer *stuffer);
+extern int s2n_stuffer_release_if_consumed(struct s2n_stuffer *stuffer);
```

Comment:
Did you mean to remove this declaration?

Reply:
Yep!